google/cloud/apigee/registry/applications/v1/diff_analytics.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package google.cloud.apigee.registry.applications.v1;
+// [START go_declaration]
+option go_package = "github.com/apigee/registry/rpc;rpc";
+// [END go_declaration]
+
+/* This proto contains the diff of a spec and its revision. Changes that are
+additions fall in the added field. Changes that are Deletions fall in the
+deleted field, and changes that modify values fall in the value_modify map
+field. */
+message Diff {
+  // Change will be displayed in format foo.bar.x
+  repeated string added = 1;
+  // Change will be displayed in format foo.bar.x
+  repeated string deleted = 2;
+
+  message ValueChange {
+    string From = 1;
+    string To = 2;
+  }
+  // Change will be displayed in format foo.bar.x {to.x from.x}
+  map <string, ValueChange> modification= 3;
```

Comment:
With the fields `added` and `deleted` I expect this to be `modified`. Alternatively `additions`, `deletions`, and `modifications` would work.